templates/rule-change-proposal.md

```diff
@@ -1,8 +1,17 @@
 **What rule do you want to change?**
 
-**Does this change cause the rule to produce more or fewer warnings?**
+**What change do you want to make? (place an "X" next to just one item)?**
```

Comment:
```suggestion
**What change do you want to make? (place an "X" next to just one item)**
```

Reply:
or  
```suggestion
**What change do you want to make (place an "X" next to just one item)?**
```

to maintain consistency at all places

Reply:
Good catch!  Fixed.